test/dnn_images_test.cpp

```diff
@@ -11,6 +11,11 @@
 
 #include <boost/thread/thread.hpp>
 
+#if CV_MAJOR_VERSION < 4
+    #define IMREAD_MODE CV_LOAD_IMAGE_COLOR
+#else
+    #define IMREAD_MODE cv::IMREAD_COLOR
```

Comment:
It is probably clearer if you rename the macro to `IMREAD_COLOR_MODE` just to keep that information in the line below.